api/lib/database/methods.js

```diff
@@ -68,6 +68,32 @@ const getValidatorsInfo = ({ hasura_url, hasura_admin_key }) => (
   )
 }
 
+const getPremiumValidators = ({
+  hasura_url,
+  hasura_admin_key
+}) => () => async () => {
+  const premiumValidators = await read({
+    hasura_url,
+    hasura_admin_key
+  })('')(`premiumValidators`, `premiumValidators`, [
+    'name',
+    'networks',
+    'rank',
+    'teamMembers',
+    'website',
+    'telegram',
+    'github',
+    'twitter',
+    'blog',
+    'contributionLinks'
+  ])
+  if (premiumValidators.length > 0) {
```

Comment:
why this check? if it is already an empty array, return that

Reply:
true, it doesn't make any sense :laughing: